webapp/snapcraft/views.py

```diff
@@ -11,8 +19,23 @@ def homepage():
 
         livestream = logic.get_livestreams()
 
+        category_list = ["Featured", "Development", "Games", "Server"]
+
+        featured_categories = []
+        for category in category_list:
```

Comment:
So I thought that what could be done was more:
On we get the featured snaps on the backend. So the first part of the caroussle displays and on the other pages of the caroussel we make an API call to the backend to get the snaps.

This would make the homepage load faster.

What do you think?